integration-tests/src/test/java/oracle/kubernetes/operator/ItPodsShutdown.java

```diff
@@ -206,23 +171,22 @@ public static void callWebApp(String testAppPath, Domain domain, boolean deployA
 
     // Send a HTTP request to keep open session
     String curlCmd = webServiceUrl.toString();
-    // LoggerHelper.getLocal().log(Level.INFO, "Send a HTTP request: " + curlCmd);
     TestUtils.checkAnyCmdInLoop(curlCmd, "Ending to sleep");
   }
 
   /**
-   * shutdown managed server.
+   * shutdown managed server and returns spent shutdown time.
```

Comment:
Please use capital to start the sentence - as per the Javadoc standard https://www.oracle.com/technetwork/java/javase/documentation/index-137868.html - in particular the section "First sentence"